test/index.js

```diff
@@ -188,6 +188,17 @@ function isHidden(filePath) {
                 // If sanitation were done, 'class' property should be removed.
                 assert.equal(React.renderToStaticMarkup(vdom), '<div><pre><code class="language-empty"></code></pre></div>');
             });
+
+            it('passes toHast options to inner toHAST() function', function() {
+                var markdown = '# Foo';
+
+                var vdom = remark().use(reactRenderer, {
+                    createElement: React.createElement,
+                    toHast: {allowDangerousHTML: true}
+                }).process(markdown, {}).contents;
```

Comment:
Here, I tested that passing an option to `toHast` configuration does not go wrong.